submissions/Vitaminvp/HTML&CSSPractice/style.css

```diff
@@ -0,0 +1,237 @@
+body {
+    font-family: "Arial", sans-serif;
+    background-color: #fff;
+    overflow-x: hidden;
+}
+ul {
+    margin: 0;
+    padding: 0;
+    list-style-type: none;
+}
+.nav{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+.nav__input,
+.nav__menu,
+.nav__menu_input,
+.nav__menu_listMore{
+    display: none;
+}
+.nav__menu{
+    position: absolute;
+    top: 50px;
+    border: 1px solid #ccc;
+    border-color: rgba(0,0,0,.2);
```

Comment:
Let's make your CSS more DRYier! 
How about using CSS variables for `rgba(0,0,0,.2)` color and other blocks?